src/index.js

```diff
@@ -17,19 +17,24 @@ export default class Scrl {
             onTick: () => {},
             friction: .7, // 1 - .3
             acceleration: .04,
+            element: window,
```

Comment:
`target` or something similar sounds better. Window isn't element after all.

Reply:
Hmm, I have thought back and forth about how to name the option. There's only [two hard things in programming](https://skeptics.stackexchange.com/questions/19836/has-phil-karlton-ever-said-there-are-only-two-hard-things-in-computer-science), right? :) 

The problem with naming it `target` would be that it could be misunderstood as the thing that should be scrolled *towards*, not the thing that should be scrolled *within*. 

How about `scrollTarget`?